regression-test/suites/mtmv_p0/test_hive_refresh_mtmv.groovy

```diff
@@ -157,9 +148,6 @@ suite("test_hive_refresh_mtmv", "p0,external,hive,external_docker,external_docke
                                     """
```

Comment:
after refresh materialized view, maybe we should add `order_qt` to check the mv result is right or not